tests/integration/cattletest/core/test_whitelist_default_registries.py

```diff
@@ -140,7 +142,7 @@ def test_registries(super_client, client, context,
 
     env = _create_stack(client)
 
-    uuid = "sim:namespace/ubuntu:{}".format(random_num())
+    uuid = "sim:namespace.com/repo/ubuntu:{}".format(random_num())
     lc_registry = {"imageUuid": uuid}
```

Comment:
yes, but `namespace` won't be the registry, it is `index.docker.io`. It is just same that `sim:rancher/agent:latest` won't have `rancher` as registry.